services/src/main/java/org/keycloak/protocol/oidc/OIDCLoginProtocolService.java

```diff
@@ -189,6 +183,16 @@ public Object token() {
         return endpoint;
     }
 
+    /**
+     * Pushed Authorization Request endpoint
+     */
+    @Path("par")
+    public Object par() {
```

Comment:
As for the way of implementing Endpoint class, It might be perferable to follow `DeviceEndpoint` and `DeviceEndpointFactory`.

Reply:
I have changed this, thank you.

Reply:
I've confirmed the change of endpoint implementation style. Thank you.
IMO, `ParEndpoint` and `ParEndpointFactory` might be in `org.keycloak.protocol.par.endpoints` as the same manner in Device Grant's endpoint and its factory. WDYT?